support/build/config-submenu.sh

```diff
@@ -92,6 +104,39 @@ else
 	exec 7>&1
 fi
 
+#
+# GENERATE LIST OF FINAL LIBRARY PATHS
+#
+LIBS=()
+for ARG_PATH in "${ARG_PATHS[@]}" "$@"; do
+	# Remove trailing slashes
+	ARG_PATH="${ARG_PATH%/}"
+
+	# Does Config.uk exist?
+	CONFIG_UK="${ARG_PATH}/Config.uk"
+	if [ ! -f "${CONFIG_UK}" ]; then
+		if [ $OPT_QUIET -ne 0 ]; then
+			echo "Could not find \"Config.uk\" under \"${ARG_PATH}\". Skipping..." 1>&2
+		fi
+		continue
+	fi
+
+	DONT_INCLUDE=1
+	for ARG_EXCLSUFFIX in "${ARG_EXCLSUFFIXES[@]}"; do
+		if [[ "${ARG_PATH}" == *"${ARG_EXCLSUFFIX}" ]]; then
+			if [ $OPT_QUIET -ne 0 ]; then
+				echo "Skipping \"${ARG_PATH}\" due to suffix matching with \"${ARG_EXCLSUFFIX}\"..." 1>&2
```

Comment:
Yes, you are right, but I think here it is for readability, so people know where the arg path starts and ends

Reply:
I see, so they are meant to be displayed. Thanks for that 👍🏼

Reply:
:+1: Yes, these quotes are for readability and will be printed. Using the curly braces for bash variables is just good practice because you can then better control `bash` where it detects the start and the end of a variable name.